tests/snuba/api/endpoints/test_group_events.py

```diff
@@ -363,3 +363,13 @@ def test_multiple_group(self):
                     six.text_type(event.event_id),
                 ]
             )
+
+    def test_boolean_feature_flag_failure(self):
+        self.login_as(user=self.user)
+        group = self.create_group()
+
+        for query in ['title:hi OR title:hello', 'title:hi AND title:hello']:
```

Comment:
So one fear I have here is that someone has OR or AND as raw....... which if so ... is going to be fairly difficult to discern. I favor using only all caps AND or OR to signify boolean values and possibly use a RAW query string instead. Wanted to get feedback on that idea before adding that as an exception to this rule.

Alternatively I could parse the string and if any BooleanSearch are in the results throw the error. That's probably the safest way.

Thoughts?

Reply:
I'd either throw an error based on there being a `BooleanSearch` in the results, or have a dynamic way to generate parts of the grammar and just completely exclude the Boolean stuff when people don't have the feature. The latter might be better so that we're not throwing weird errors at people while we develop this feature.

Reply:
^ The latter there could also be used in Issue Search so that we don't need to throw an error